nixos/modules/services/web-apps/nextcloud.nix

```
@@ -254,6 +254,14 @@ in {
       '';
     };
 
+    fastcgiTimeout = mkOption {
+      type = types.int;
+      default = 120;
+      description = ''
```

```suggestion
      description = mdDoc ''
```
writing new module documentation as docbook is discouraged.

---

but, there's no docbook in it?

---

it'll be parsed as docbook regardless, and the more options we can *not* parse as docbook when they're introduced the easier the migration away from docbook will be